<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setup_dialog</class>
 <widget class="QDialog" name="setup_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>246</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Настройки</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/icons/logo</normaloff>:/icons/logo</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Построение ракурса</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Высота</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Ширина</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Размер стороны текстуры</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Размер стороны сетки текстур</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Детализация</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Палитра</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLineEdit" name="height_edit"/>
            </item>
            <item>
             <widget class="QLineEdit" name="width_edit"/>
            </item>
            <item>
             <widget class="QLineEdit" name="texture_size_edit"/>
            </item>
            <item>
             <widget class="QLineEdit" name="texture_per_row_edit"/>
            </item>
            <item>
             <widget class="QComboBox" name="accuracy_box">
              <item>
               <property name="text">
                <string>Высокая</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Низкая</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="palette_box">
              <item>
               <property name="text">
                <string>211 style</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Akinin's style</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="is_polygon_as_a_line_box">
          <property name="text">
           <string>Тонкие линии</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="apply_button">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Применить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="reset_button">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Сбросить</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>height_edit</tabstop>
  <tabstop>width_edit</tabstop>
  <tabstop>texture_size_edit</tabstop>
  <tabstop>texture_per_row_edit</tabstop>
  <tabstop>accuracy_box</tabstop>
  <tabstop>palette_box</tabstop>
  <tabstop>is_polygon_as_a_line_box</tabstop>
  <tabstop>apply_button</tabstop>
  <tabstop>reset_button</tabstop>
 </tabstops>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
